API/NoDocs/Microsoft.AspNetCore.Telemetry/HttpLoggingServiceExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Assembly 'Microsoft.AspNetCore.Telemetry.Middleware'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Telemetry;
```

Comment:
@davidfowl @noahfalk

Is *.Telemetry the right namespace for this stuff?

Reply:
The existing ASP.NET logging stuff is in Microsoft.AspNetCore.HttpLogging, so that might be a better choice for this. Thoughts @davidfowl @Tratcher?

Reply:
These are all extension methods on IServiceCollection and belong in the Microsoft.Extensions.DependencyInjection namespace with it.

Reply:
I assume the scope here is all of the types that use Microsoft.AspNetCore.Telemetry, not only the HttpLoggingServiceExtensions class. I agree that for service extensions specifically the M.E.DI namespace is appropriate. James already flagged an issue on that for all the service collection extensions.

Reply:
Non-extension types should use a namespace derived from the assembly name. Naming the assembly is a different discussion.

Reply:
Done